src/System.Runtime.WindowsRuntime.UI.Xaml/tests/Configurations.props

```diff
@@ -1,7 +0,0 @@
-﻿<Project DefaultTargets="Build">
-  <PropertyGroup>
-    <BuildConfigurations>
```

Comment:
Just curious. Is it intentional that we're not testing on .NET Core? The implementation indeed supports netcoreapp.

Reply:
this is a mistake. thanks!